File: 2.0/csharp/calls/getCallBroadcastCalls.cs
using CallfireApiClient;
using CallfireApiClient.Api.Common.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new GetBroadcastCallsTextsRequest
        {
            Id = 11646003,
            BatchId = 5500030002,
            Offset = 0,
            Limit = 10,
            Fields = "items(fromNumber,toNumber,state,modified)"
        };
        var calls = client.CallBroadcastsApi.GetCalls(request);
    }
}

File: 2.0/java/calls/getCallBroadcastCalls.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.callstexts.model.Call;
import com.callfire.api.client.api.campaigns.model.request.FindBroadcastCallsRequest;
import com.callfire.api.client.api.common.model.Page;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        FindBroadcastCallsRequest request = FindBroadcastCallsRequest.create()
            .id(11646003L)
            .batchId(5500030002L)
            .offset(0L)
            .limit(10L)
            .fields("items(fromNumber,toNumber,state,modified)")
        .build();
        Page<Call> calls = client.callBroadcastsApi().findCalls(request);
    }
}
